test/includes/net.sh

```diff
@@ -8,6 +8,7 @@ local_tcp_port() {
     while true; do
         port=$(shuf -i 10000-32768 -n 1)
         nc -l 127.0.0.1 "${port}" >/dev/null 2>&1 &
```

Comment:
Maybe it'd be better to switch to `socat` here as well and give it a timeout.

Reply:
Using `socat` actually seems to fix the issue altogether. No more `sleep` needed.